Library/Homebrew/formula_creator.rb

```diff
@@ -134,7 +150,7 @@ def install
                                   "--prefix=\#{prefix}"
             # system "cmake", ".", *std_cmake_args
         <% end %>
-        <% if mode != :meson and mode != :go %>
+        <% if mode != :meson and mode != :go and mode != :python %>
```

Comment:
Maybe just `if autotools || cmake` at this point?

Reply:
Yes, this definitely looks better. Committed.